rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc

```diff
@@ -0,0 +1,72 @@
+= Activity Library
+
+The Activity Library enables you to reuse certain activities that repeat during the automation of business processes, a login for example, and avoid creating the same workflow multiple times.
+
+A particular case is the Start event (Trigger). Although it is not an activity in the strict sense, a workflow is still assigned to this item, which initially starts the process. You can upload this event to the Activity Library. 
```

Comment:
**issue**: Trigger Workflow does no longer exist. This was a mechanism in Legacy X1, but is no longer used in MuleSoft RPA